src/main/java/org/gwtproject/i18n/processor/GwtLocaleFactoryImpl.java

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright 2009 Google Inc.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.gwtproject.i18n.processor;
+
+import org.gwtproject.i18n.processor.GwtLocaleImpl;
+import org.gwtproject.i18n.shared.GwtLocale;
+import org.gwtproject.i18n.shared.GwtLocaleFactory;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+/**
+ * Creates server-side GwtLocale instances.  Thread-safe.
+ */
+public class GwtLocaleFactoryImpl implements GwtLocaleFactory {
```

Comment:
i know we've got a bazillion files here, but could you tag the ones that are just being moved from another repo, which ones are your own entirely?

Reply:
maybe this isnt helpful after all, having finished scanning the whole diff